app/views/incomes/estimatedPayment/update/checkYourAnswers.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers._
+@import includes.ptaHeader
+@import uk.gov.hmrc.urls.Link
+@import includes._
+@import uk.gov.hmrc.tai.viewModels.income.estimatedPay.update.CheckYourAnswersViewModel
+
+@(viewModel: CheckYourAnswersViewModel, employerName: String)(implicit request: Request[_], messages: Messages, user: controllers.auth.TaiUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+
+@header = @{
+ptaHeader(
+displayBackLink = true,
+mainHeadingText = Messages("tai.checkYourAnswers"),
+preHeadingText = Messages("tai.incomes.edit.preHeading", employerName))
+}
+
+@main(
+title = Messages("tai.checkYourAnswers"),
+user = Some(user),
+pageHeader = Some(header),
+articleClasses = None,
+gaCustomTitle = None
+){
+
+
+<div class="section">
+    @includes.checkYourAnswers(
+    title = None,
+    longQuestions = true,
+    confirmationLines = viewModel.journeyConfirmationLines
```

Comment:
We're not passing through any GA actions for the clicks. Should we follow up on if we need any?

Reply:
This will come in on a separate ticket.